backend/internal/user/handler/userhandler.go

```diff
@@ -331,6 +372,8 @@ func handleError(w http.ResponseWriter, logger *log.Logger, svcErr *serviceerror
 		switch svcErr.Code {
 		case constants.ErrorMissingUserID.Code:
 			statusCode = http.StatusNotFound
+		case constants.ErrorUserNotFound.Code:
+			statusCode = http.StatusNotFound
 		case constants.ErrorOrganizationUnitNotFound.Code:
 			statusCode = http.StatusNotFound
```

Comment:
```suggestion
        case constants.ErrorMissingUserID.Code, constants.ErrorUserNotFound.Code, constants.ErrorOrganizationUnitNotFound.Code:
            statusCode = http.StatusNotFound
```

Can combine multiple case statements with the same body

Reply:
Fixed with https://github.com/asgardeo/thunder/compare/8fe025d2136a7db365d70d44d136f7debc9a9c91..50c21d7afd511aa6b76090023a46fbace6bab68f